<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuProfile</class>
 <widget class="QDialog" name="StuProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="join">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Join Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>590</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>120</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit Project</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>467</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_47">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Student Number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="id_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>105</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Project</string>
   </property>
  </widget>
  <widget class="QLabel" name="stuname">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Student's profile</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>431</width>
     <height>486</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_48">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>GeneralCodingAbility                 </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gcode_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_43">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_49">
      <item>
       <widget class="QLabel" name="label_59">
        <property name="text">
         <string>PrimaryCodingAbility                 </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pcode_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_44">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_50">
      <item>
       <widget class="QLabel" name="label_60">
        <property name="text">
         <string>WritingSkills                                  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="write_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_45">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_51">
      <item>
       <widget class="QLabel" name="label_61">
        <property name="text">
         <string>DeadlinePunctuality                   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="punc_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_46">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_52">
      <item>
       <widget class="QLabel" name="label_62">
        <property name="text">
         <string>LeadershipAbility                        </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lead_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_47">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_53">
      <item>
       <widget class="QLabel" name="label_63">
        <property name="text">
         <string>WorkEthic                                      </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Ethic_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_48">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_54">
      <item>
       <widget class="QLabel" name="label_64">
        <property name="text">
         <string>TeamWorkingAbility                  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="teamwork_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_49">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_55">
      <item>
       <widget class="QLabel" name="label_65">
        <property name="text">
         <string>PresentationSkills                       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="presentation_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_50">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_56">
      <item>
       <widget class="QLabel" name="label_66">
        <property name="text">
         <string>AbilityToManageStress             </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="stress_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_51">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_57">
      <item>
       <widget class="QLabel" name="label_67">
        <property name="text">
         <string>ClassAttendance                         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="att_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_52">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_58">
      <item>
       <widget class="QLabel" name="label_68">
        <property name="text">
         <string>DistanceFromCampus               </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="distance_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_53">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_59">
      <item>
       <widget class="QLabel" name="label_69">
        <property name="text">
         <string>AvailableTime                              </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="time_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_54">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_60">
      <item>
       <widget class="QLabel" name="label_70">
        <property name="text">
         <string>Overall CGPA                                </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cgpa_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_55">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_61">
      <item>
       <widget class="QLabel" name="label_71">
        <property name="text">
         <string>GradeOfPrerequisiteCourse     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pre_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_56">
        <property name="text">
         <string>Preferred</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="currentList">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Project list</string>
   </property>
  </widget>
  <widget class="QListView" name="projectList">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>221</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
